frontend/actions.js

```diff
@@ -26,67 +26,101 @@ export default ({setState, getState}) => {
       optionalArgsObj
     )
 
-  const loadWalletFromMnemonic = async (state, mnemonic) => {
+  const loadWallet = async (state, {cryptoProvider, secret}) => {
     setState(loadingAction(state, 'Loading wallet data...'))
+    setState({walletLoagingError: undefined})
+    switch (cryptoProvider) {
+      case 'trezor':
+        try {
+          wallet = await Cardano.CardanoWallet({
+            cryptoProvider: 'trezor',
+            config: CARDANOLITE_CONFIG,
+          })
+        } catch (e) {
+          return setState({
+            loading: false,
+            walletLoagingError:
+              'Trezor authentication rejected! Check your Trezor firmware version.',
```

Comment:
Just wanted  to point out that Trezor popup will show error message if you are using trezor with firmware which is not supported.

![screenshot from 2018-06-28 10-39-06](https://user-images.githubusercontent.com/2537379/42023227-9fa57422-7abf-11e8-9525-99be72dd60f3.png)

Reply:
ok, then this is temp, 
currently we do not ask for firmware version, since there is no compatible version. 
Communication just fails... . (no info in popup)